server/core/src/main/java/dev/slimevr/tracking/trackers/udp/UDPPacket.kt

```diff
@@ -320,6 +320,19 @@ data class UDPPacket21UserAction(var type: Int = 0) : UDPPacket(21) {
 	}
 }
 
+class UDPPacket22FeatureFlags(
```

Comment:
Is there a reason why it's a separate packet instead of putting it into handshake or sensor info?

Reply:
Data exchange is bidirectional here, adding extra data to server->firmware handshake breaks owotrack, though I didn't dig into the handshake format too much.